src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -31,9 +31,9 @@ public class TelegramBotClient : ITelegramBotClient
 
         private static readonly Update[] EmptyUpdates = { };
 
-        private const string BaseUrl = "https://api.telegram.org/bot";
+        private string BaseUrl = "https://api.telegram.org/bot";
 
-        private const string BaseFileUrl = "https://api.telegram.org/file/bot";
+        private string BaseFileUrl = "https://api.telegram.org/file/bot";
```

Comment:
Man you have to change fields and mark them as readonly. You initizilithation BaseFileUrl and BaseUrl  in a constructor and when declare them.